common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -779,7 +779,10 @@ def _load_items(self, course_entry, block_keys, depth=0, **kwargs):
             self._add_cache(course_entry.structure['_id'], runtime)
             self.cache_items(runtime, block_keys, course_entry.course_key, depth, lazy)
 
-        return [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
+        blocks = [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
+        for block in blocks:
+            block.course_version = course_entry.course_key.version_guid
```

Comment:
I guess it makes me a little nervous to have another thing that leaks through the modulestore abstraction like this, since people might assume it exists in both. Could we also add it to the old mongo modulestore, and just have it set to `None` there, so at least the attribute exists in both places?